config/file_resolver.go

```diff
@@ -42,8 +42,11 @@ func NewRelativeResolver(paths ...string) FileResolver {
 
 	copy(pathList, paths)
 
-	// add the current cwd
-	if cwd, err := os.Getwd(); err == nil {
+	if appRoot := os.Getenv(EnvironmentPrefix() + _root); appRoot != "" {
```

Comment:
Note that environment prefix for uber services won't be "APP". I believe you want _root to be just "APP_ROOT" and not use EnvironmentPrefix()